package/ebs/rancher-ebs

```diff
@@ -35,7 +35,8 @@ wait_volume_transition() {
         print_error "Failed volume ${VOLUME_ID} transition, expected end state is: ${end_state}, got ${current_state}"
     fi
     if [ $retriess -ge 10 ]; then
-        exit 2
+        print_error "Failed volume ${VOLUME_ID} transition, expected end state is: ${end_state}, got ${current_state} after ${retries} retries"
```

Comment:
print_error will exit a different code https://github.com/rancher/storage/blob/master/package/common/common.sh#L85-L89.

Reply:
Swapped first print_error with log_error and second print_error with log_warn (which seemed appropriate for the context).